app/controllers/api/v1/statuses/mutes_controller.rb

```diff
@@ -22,6 +22,16 @@ def destroy
     render json: @status, serializer: REST::StatusSerializer
   end
 
+  def clear_notifications
+    current_account.notifications.where(activity_type: 'Mention').joins(mention: :status) \
+                   .where(status: { in_reply_to_id: params[:status_id] }).destroy_all
+    current_account.notifications.where(activity_type: 'Status').joins(:status) \
+                   .where(status: { reblog_of_id: params[:status_id] }).destroy_all
+    current_account.notifications.where(activity_type: 'Favourite').joins(:favourite) \
+                   .where(favourite: { status_id: params[:status_id] }).destroy_all
```

Comment:
Please remove the extraneous `\` characters.

```suggestion
    current_account.notifications.where(activity_type: 'Mention').joins(mention: :status)
                   .where(status: { in_reply_to_id: params[:status_id] }).destroy_all
    current_account.notifications.where(activity_type: 'Status').joins(:status)
                   .where(status: { reblog_of_id: params[:status_id] }).destroy_all
    current_account.notifications.where(activity_type: 'Favourite').joins(:favourite)
                   .where(favourite: { status_id: params[:status_id] }).destroy_all
```